src/main/java/org/spongepowered/api/item/properties/DamageAbsorptionProperty.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of Sponge, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered.org <http://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+package org.spongepowered.api.item.properties;
+
+/**
+ * Represents the amount of damage the item will absorp when equipped. The
+ * damage absorbed can be dependent on the type of damage, equipment type, and
```

Comment:
> Armor absorbs a percentage of damage, not a fixed amount.

True. There is a fixed default value that is used with some function to calculate the absorbed damage.

> Isn't this some kind of implementation restriction that should not be enforced by the API?

The point of ItemProperty is to represent immutable data that relates to the item itself. A `DIAMOND_CHESTPLATE` will always have a maximum durability, just like how a `DIAMOND_CHESTPLATE` will always have a specific percentage of damage reduction. Can events change this damage reduction? Absolutely! Does that change the fact that it's an ItemProperty? No.

> Maybe a concept like AttributeModifiers would work best here.

Not in the scope of this PR as Attributes is being handled in #403.